main.py

```diff
@@ -1,13 +1,14 @@
-from logic import *
-
-#Start your program here
-
-a = 1
-
-def b():
-    global a
-    a += 2
-    print( " " + str(a))
-
-b()
-print(a)
\ No newline at end of file
+sum_list = []
+
+while True:
+    number_1 = int(input ("What is number 1"))
+    number_2 = int(input ("What is number 2"))
+    print(number_1 + number_2)
+    answer = (number_1 + number_2)
+    if answer != 0:
+        sum_list.append(answer)
```

Comment:
Please link me the documentation/article from where you learned this. :) Using new concepts in class is awesome, but I want  to make sure that you actually learned it, instead of having someone just tell you about it.

Reply:
https://www.tutorialspoint.com/python/comparison_operators_example.htm is where I found != and I used it